### PR TITLE
Spelling fix

### PR DESCRIPTION
**Pull request by TessFerrandez**
Girl scout cleanup - fixing spelling mistakes and rewording sentences that use non-existing words throughout the repo

### CODE REVIEW

**Review by zoekdestep — APPROVED** [review]

File: observability/correlation-id.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## The Need
 
-In a distributed system architecture (microservice architecture), it is highly difficult to understand a single end to end customer transaction flow through the various components.
+In a distributed system architecture (micro service architecture), it is highly difficult to understand a single end to end customer transaction flow through the various components.
```
Comment (fnocera) [inline observability/correlation-id.md:5]: I would argue that "microservice" as one word is how it is most often referred to....

File: design-reviews/decision-log/readme.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ An architecture decision record has the structure
 
     Example:
 
-    > *Due to the microservices design of the platform, we need to ensure consistency of logging throughout each service so tracking of usage, performance, errors etc. can be performed end-to-end. A single logging/monitoring framework should be used where possible to achieve this, whilst allowing the flexibility for integration/export into other tools at a later stage. The developers should be equipped with a simple interface to log messages and metrics.*
+    > *Due to the micro services design of the platform, we need to ensure consistency of logging throughout each service so tracking of usage, performance, errors etc. can be performed end-to-end. A single logging/monitoring framework should be used where possible to achieve this, whilst allowing the flexibility for integration/export into other tools at a later stage. The developers should be equipped with a simple interface to log messages and metrics.*
```
Comment (fnocera) [inline design-reviews/decision-log/readme.md:47]: As with the other comment would leave as "microservices"

File: design-reviews/readme.md [hunk 3]
```diff
@@ -62,14 +62,14 @@ The dev crew should always participate in all design review sessions
...
-- Customer Subject Matter Experts (SME's)
+- Customer Subject Matter Experts (SME)
 - Senior Leadership
 
 ## Facilitation Guidance
 
 ### Sync Design Reviews via in-person / virtual meetings
 
-Joint meetings with dev crew, subject matter experts (SME's) and customer engineers
+Joint meetings with dev crew, subject matter experts (SME) and customer engineers
```
Comment (fnocera) [inline design-reviews/readme.md:72]: Maybe "SMEs"

File: engineering-feedback/feedback-guidance.md [hunk 4]
```diff
@@ -43,17 +43,17 @@ Select one of the following to describe the lifecycle stage of the engagement th
 
 Describe the impact to the customer and engagement that this feedback implies.
 
-### Timeframe
+### Time frame
```
Comment (fnocera) [inline engineering-feedback/feedback-guidance.md:46]: So interesting, I have always written it as one word but I think you are correct!

**Review by fnocera — APPROVED** [review]
LGTM

**Comment by TessFerrandez** [comment]
Thank you @zoekdestep for reviewing - I believe you need to get contributor rights to count as a reviewer - @fnocera can help you

**Pull request merged**